.github/workflows/merge-flow.yaml

```diff
@@ -159,6 +159,19 @@ jobs:
       - name: Continue after merge conflict
         if: ${{ steps.merge.outputs.MERGE_CONFLICT == 'true' }}
         run: git merge --continue
+      - name: Add VERSION file
+        run: |
+          # All tags use the vX.Y.Z format currently.
+          version_from_tag=$(echo ${{ steps.upstream.outputs.release }} | sed -e "s/^v//")
+          if [ -f VERSION ]; then
+            if [ "$version_from_tag" != "$(cat VERSION)" ];then
+              echo "::error:: tag version doesn't correspond to VERSION"
```

Comment:
Maybe log the two mismatched version here for easier debugging?

Reply:
done.